.asf.yaml

```diff
@@ -0,0 +1,36 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories
+
+github:
+  description: "Apache Beam is a unified programming model for Batch and Streaming"
+  homepage: https://beam.apache.org/
+  labels:
+    - python
+    - java
+    - big-data
+    - beam
+
+  enabled_merge_buttons:
+    squash: true
+    merge:  true
+    rebase: true
+
+notifications:
```

Comment:
@iemejia does this match?

Reply:
Yes they are good. The only outcome I don't like of this change by INFRA is that we have two places to configure mailing lists notifications because other mailing lists (e.g. issues@) get data from JIRA and not github and we cannot configure this with this file at the moment, but well maybe that will be done in the future.

Reply:
@iemejia Do you know if there's a separate settings file somewhere for jira notifications?

Reply:
Nop the canonical reference so far it is the .asf.yaml file https://s.apache.org/asfyaml-notify
Maybe worth to create a JIRA to request this.